.github/workflows/release.yml

```diff
@@ -184,10 +184,8 @@ jobs:
           JIRA_SUMMARY: ${{ env.jira_summary }}
```

Comment:
These code changes involve updating the RELEASE_NOTES variable to include Ticket Details with JIRA_SUMMARY and JIRA_DESCRIPTION. This change adds the JIRA ticket details to the release notes generated during the workflow execution.

The code now appends the JIRA ticket details to the RELEASE_NOTES variable by including the JIRA_SUMMARY and JIRA_DESCRIPTION variables. This will provide more information in the release notes about the JIRA ticket related to the release, enhancing the overall visibility of the release process.